modules/ztp-pre-staging-downloading-artifacts.adoc

```diff
@@ -0,0 +1,367 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-pre-staging-tool.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-downloading-images_{context}"]
+= Downloading the images
+
+The {factory-prestaging-tool} allows you to download the following images to your partitioned server:
+
+* {product-title} images
+* Operator images that are included in the distributed unit (DU) profile for 5G RAN sites
+* Operator images from disconnected registries
+
+[NOTE]
+====
+The list of available Operator images can vary in different {product-title} releases.
+====
+
+[id="ztp-preparing-ocp-images_{context}"]
+== Preparing to download the {product-title} images
+
+To download the {product-title} container images, you need the current version of {rh-rhacm-first} to provision the {sno}.
```

Comment:
,you need to know the RHACM version running in the hub cluster that is going to provision and manage the single-node OpenShift.